test/index.spec.ts

```diff
@@ -14,14 +14,16 @@ import {
   DEFAULT_EMAIL,
   sessionStorage,
 } from './utils'
+import { Session } from '@remix-run/server-runtime'
```

Comment:
Is `Session` used only as a type? If so, it should be imported as a type, and I think it wasn't. No worries though, I will look into it right now.

Also, are `SendTOTPOptions`, `TOTPData`, and `TOTPStrategy` used as types too, right?

I think they were not imported as types before, right? In that case, it would be my mistake, or I'm probably being wrong about it. _(I'll have to revisit the code in my editor, that would help me a bit)._

Reply:
`Session`, `SendTOTPOptions`, and `TOTPData` are types and I'll import them as such in the next revision. `TOTPStrategy` is a class.